models/surgical_tool_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,77 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.1.0",
+    "changelog": {
+        "0.1.0": "complete the first version model package",
+        "0.0.1": "initialize the model package structure"
+    },
+    "monai_version": "1.0.0",
+    "pytorch_version": "1.12.0",
+    "numpy_version": "1.22.4",
+    "optional_packages_version": {
+        "nibabel": "4.0.1",
+        "pytorch-ignite": "0.4.9"
+    },
+    "task": "surgical tool segmentation",
+    "description": "A pre-trained binary segmentation model for surgical tool segmentation",
```

Comment:
Thanks @binliunls , this bundle looks good to me.
Hi @Nic-Ma and @wyli , since the dataset is private, do you think it is better to add some figures to visualize what kinds of tools can be recognized based on this model? For example, do inference on some similar and public datasets (such as [this bundle](https://github.com/Project-MONAI/model-zoo/tree/dev/models/ventricular_short_axis_3label) does).

Reply:
Interesting idea!
@binliunls Could you please help add inference example with the public dataset in the video tutorial?

Thanks in advance.

Reply:
Hi @Nic-Ma , @yiheng-wang-nv 
     I have added the EndoVis 2017 dataset to the readme file.